cartography/intel/snipeit/util.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from typing import Any
+from typing import Dict
+
+import requests
+
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+# Connect and read timeouts of 60 seconds each; see https://requests.readthedocs.io/en/master/user/advanced/#timeouts
+_TIMEOUT = (60, 60)
+
+
+@timeit
+def call_snipeit_api(api_and_parameters: str, base_uri: str, token: str) -> Dict[str, Any]:
```

Comment:
Non-block to consider for the future: use something like backoff to do autoretry


Example
```
import backoff

@backoff.on_exception(
    wait_gen=backoff.expo,
    exception=RuntimeError,
    max_tries=5,
    on_backoff=backoff_handler,
)
```

Reply:
Tried adding the `backoff` decorator as suggested and got below error
```cartography/intel/snipeit/util.py:16: error: Untyped decorator makes function "call_snipeit_api" untyped  [misc]```